Dockerfile

```diff
@@ -1,13 +1,24 @@
 # Mozilla Kinto server
-FROM python:3.5
+FROM debian:sid
```

Comment:
stackbrew/debian    sid                 e560b1d2575a        8 days ago          117.2 MB
debian              sid                 e560b1d2575a        8 days ago          117.2 MB

Look at the IMAGE ID, they are the exact same images :)